ports/ziiaol/port.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "ziiaol.zip",
+  "items": [
+    "Zelda II: Enhanced.sh",
+    "ziiaol"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Zelda II: Enhanced",
+    "porter": [
+      "Jeod"
+    ],
+    "desc": "Zelda II Enhanced adds a ton of QoL improvements, widescreen, new areas, and new collectibles. The game is arguably much easier to play through.",
+    "desc_md": "Zelda II Enhanced adds a ton of QoL improvements, widescreen, new areas, and new collectibles. The game is arguably much easier to play through.",
+    "inst": "Download the .zip from https://hoverbat.itch.io/ziiaol and put in the gamedata folder.",
+    "inst_md": "Download the .zip from https://hoverbat.itch.io/ziiaol and put in the gamedata folder.",
```

Comment:
If that markdown section is the same as the normal one we just use one field. It's just to make the details page prettier with the md links. Or if the instructions are big we can just put small text one in the normal field and go crazy on the md field